views/error-gae-auth.html.twig

```diff
@@ -3,6 +3,6 @@
 {% block body %}
     <div class="alert alert-danger">
         <span class="glyphicon glyphicon-thumbs-down"></span>
-        <strong>Access denied!</strong> You are not logged into Google, please <a href="{{ loginUrl }}">try again</a>.
+        <strong>Access denied!</strong> You are not logged in, please <a href="{{ path('mockLogin') }}">Login</a>.
```

Comment:
Since the link is part of a sentence ("please login"), I suggest changing the case:

```suggestion
        <strong>Access denied!</strong> You are not logged in, please <a href="{{ path('mockLogin') }}">login</a>.
```